core/src/com/unciv/logic/civilization/QuestManager.kt

```diff
@@ -320,12 +374,56 @@ class QuestManager {
     private fun getScoreForQuest(assignedQuest: AssignedQuest): Int {
         val assignee = civInfo.gameInfo.getCivilization(assignedQuest.assignee)
         return when (assignedQuest.questName) {
+            "Contest Culture" -> assignee.policies.totalAccumulatedCulture - assignedQuest.data1.toInt()
```

Comment:
Are you sure this was in Vanilla civ? I'm pretty sure it wasn't.

Reply:
Nope, you are right. No Contest quest was in Vanilla.. they were introduced in G&K. Should I remove them?

Reply:
Yes, and the tech contest as well. We're sticking to Civ V features ;)